test/test_linalg.py

```diff
@@ -1124,6 +1084,7 @@ def test_eigvalsh_errors_and_warnings(self, device, dtype):
             with self.assertRaisesRegex(RuntimeError, "tensors to be on the same device"):
                 torch.linalg.eigvalsh(t, out=out)
 
+<<<<<<< HEAD
```

Comment:
Git conflict markers have leaked into the PR.

Reply:
Oooops. Thanks for the catch, fixed now.